File: js/task2b/S2_FOR_04.js
function countOccurrences(values) {
  const counts = {};

  for (let value of values) {
      if (counts[value] === undefined) {
          counts[value] = 1;
      } else {
          counts[value]++;
      }
  }

  return counts;
}

console.log(countOccurrences([1, 2, 2, 3]));       
console.log(countOccurrences(['a', 'b', 'a']));    
console.log(countOccurrences([]));                 

File: python/task2b/S2_FOR_04.py
def countOccurrences(values):
  counts = {}
  for v in values:
      counts[v] = counts.get(v, 0) + 1
  return counts

print(countOccurrences([1, 2, 2, 3]))
print(countOccurrences(['a', 'b', 'a']))
print(countOccurrences([]))
